src/main/java/duke/ui/UI.java

```diff
@@ -0,0 +1,55 @@
+package duke.ui;
+
+import duke.task.TaskList;
+
+public class UI {
```

Comment:
Although UI is short form of user interface, but base on coding standard, class name should be in PascalCase. Maybe Ui is better?

Reply:
Indeed it should follow PascalCase. I will make the changes.